src/sass/app/metatrader.scss

```diff
@@ -211,13 +211,13 @@
                 border-bottom: none;
             }
         }
-        .add_more_servers {
+        .add_region {
             position: absolute;
-            bottom: 8px;
-            right: 8px;
+            bottom: 16px;
+            right: 21px;
 
             span {
-                padding: 6px 8px;
+                padding: 9px;
```

Comment:
Nit, can we use `rem` instead of `px`

Reply:
@matin-binary I believe on binary static we do not define the constant size of `rem` unit, therefore, we use pixels across all the styles. Let's keep it this way for consistency.